aminator/plugins/provisioner/apt.py

```diff
@@ -113,15 +123,19 @@ def deb_query(package, queryformat, local=False):
             cmd = 'dpkg-query -W'.split()
             cmd.append('-f={0}'.format(queryformat))
         cmd.append(package)
-        return monitor_command(cmd)
+        deb_query_result = monitor_command(cmd)
+        if not deb_query_result.success:
+            log.debug('failure:{0.command} :{0.std_err}'.format(deb_query_result.result))
+        return deb_query_result
 
     @cmdsucceeds("aminator.provisioner.apt.apt_get_update.count")
     @cmdfails("aminator.provisioner.apt.apt_get_update.error")
     @timer("aminator.provisioner.apt.apt_get_update.duration")
     @retry(ExceptionToCheck=AptProvisionerUpdateException, tries=5, delay=1,
            backoff=0.5, logger=log)
-    def apt_get_update(self):
-        self.apt_get_clean()
+    @classmethod
+    def apt_get_update(cls):
```

Comment:
apt-get update made more sense as a class method since it doesn't rely on internal state